Formula/netcdf-mpi.rb

```diff
@@ -0,0 +1,153 @@
+class NetcdfMpi < Formula
+  desc "Libraries and data formats for array-oriented scientific data"
+  homepage "https://www.unidata.ucar.edu/software/netcdf"
+  url "https://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-c-4.8.0.tar.gz"
+  mirror "https://www.gfd-dennou.org/arch/netcdf/unidata-mirror/netcdf-c-4.8.0.tar.gz"
+  sha256 "679635119a58165c79bb9736f7603e2c19792dd848f19195bf6881492246d6d5"
+  license "BSD-3-Clause"
+  revision 1
+  head "https://github.com/Unidata/netcdf-c.git", branch: "main"
+
+  livecheck do
+    url "https://www.unidata.ucar.edu/downloads/netcdf/"
+    regex(/href=.*?netcdf-c[._-]v?(\d+(?:\.\d+)+)\.t/i)
+  end
+
+  bottle do
+    sha256 cellar: :any,                 arm64_big_sur: "0c50d7fa0859656c1eb01da3de8be9eba53189c6038a901da6679746915fdb32"
+    sha256 cellar: :any,                 big_sur:       "71057c8d7e1ebb521973d2f5841f0b1d093df47cbd930091ee0add6f58e91f53"
+    sha256 cellar: :any,                 catalina:      "e75f036bc6f9a8979ab702b9293c5eaede59d7721f1e05ce62ff9b87cb224398"
+    sha256 cellar: :any,                 mojave:        "15488b7028f4b086ebe2e540cbbfd2e8c6109593ed33a14af467addc84faf9f5"
+    sha256 cellar: :any_skip_relocation, x86_64_linux:  "e2ad1d7bc3c8aecd74b25592f154dc7290b6c607199212baf3e96a087025b0ef"
+  end
+
+  depends_on "cmake" => :build
+  depends_on "gcc"
+  depends_on "hdf5-mpi"
+  depends_on "open-mpi" => [:cc, :cxx, :fc]
+
+  uses_from_macos "curl"
+  conflicts_with "netcdf", because: "netcdf-mpi is a variant of netcdf, one can only use one or the other"
+
+  resource "cxx" do
+    url "https://www.unidata.ucar.edu/downloads/netcdf/ftp/netcdf-cxx4-4.3.1.tar.gz"
+    mirror "https://www.gfd-dennou.org/arch/netcdf/unidata-mirror/netcdf-cxx4-4.3.1.tar.gz"
+    sha256 "6a1189a181eed043b5859e15d5c080c30d0e107406fbb212c8fb9814e90f3445"
+  end
+
+  resource "fortran" do
+    # Source tarball at official domains are missing some configuration files
+    # Switch back at version bump
+    url "https://github.com/Unidata/netcdf-fortran/archive/refs/tags/v4.5.3.tar.gz"
+    sha256 "c6da30c2fe7e4e614c1dff4124e857afbd45355c6798353eccfa60c0702b495a"
+  end
+
+  def install
+    ENV.deparallelize
+    common_args = std_cmake_args << "-DBUILD_TESTING=OFF "
```

Comment:
```suggestion
    common_args = std_cmake_args
```
`BUILD_TESTING=OFF` is already part of `std_cmake_args`.